pkg/rehearse/jobs.go

```diff
@@ -110,24 +110,60 @@ func makeRehearsalPresubmit(source *prowconfig.Presubmit, repo string, prNumber
 	return &rehearsal, nil
 }
 
-func filterJobs(changedPresubmits map[string][]prowconfig.Presubmit, allowVolumes bool, logger logrus.FieldLogger) config.Presubmits {
-	ret := config.Presubmits{}
+func makeRehearsalPeriodic(source *prowconfig.Periodic, prNumber int) (prowconfig.Periodic, error) {
```

Comment:
My comment on refs seems to have been lost here

Reply:
oh. I will address this.